Bottom: 3473a40e0056c1f3264c3c5be0213424ec97924c
Top:    a23d8585160d52d515b3e735117844e06b92388d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-04 03:04:39 +0100

hid/gtk: Remove debugging printf accidentally committed


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index d20a90f..ae8cfa4 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -482,7 +482,6 @@ cancel_tooltip_update (GHidPort *out)
 {
   if (out->tooltip_update_timeout_id)
     {
-      printf ("cancel_tooltip_update called to remove timeout id %i\n", out->tooltip_update_timeout_id);
       g_source_remove (out->tooltip_update_timeout_id);
       out->tooltip_update_timeout_id = 0;
     }
